### ISSUE TITLE
Add health check to the Docker image

### ISSUE THREAD

**Issue by GergelyKalmar** [validator/validator#1588]
**Is your feature request related to a problem? Please describe.**
We would like to know when the service is ready to respond to requests. This would be particularly useful in cases where the service is started automatically (e.g. in automated test suites) – we would like to wait until the initialization is completed before sending requests for checks.

**Describe the solution you'd like**
Optimally the Dockerfile would already come with a well-defined healtcheck (see https://docs.docker.com/engine/reference/builder/#healthcheck).

**Describe alternatives you've considered**
We tried to add a health check in our Docker Compose service definition, however, as the image is extremely slim we weren't able to execute a GET request against the service to check if it responds (curl is not available, wget isn't available either, the /dev/tcp/ approach doesn't work because bash is not available). One alternative would be to install at least wget inside the image so that we can ping the service endpoint for health checking.

**Comment by sideshowbarker** [comment]
OK, in https://github.com/validator/validator/commit/7b81adae4c9733e60bec3595c99ccd381e717636 I added a `HEALTHCHECK` and then pushed new image.

The `HEALTHCHECK` looks like this:

```dockerfile
HEALTHCHECK --interval=30s --timeout=3s --start-period=10s --retries=3 \
    CMD wget --no-verbose --tries=1 --spider http://localhost:8888/ || exit 1
```

If that’s not what you had in mind — or if you have suggestions for further refinements — lemme know.

**Comment by GergelyKalmar** [comment]
Awesome! Checked it against `latest` and it works fine. Could this be released as part of a versioned tag? We don't use `latest` to ensure the stability of our test suites.

**Comment by sideshowbarker** [comment]
For now if you need a stable version, we actually already have those here:

https://github.com/validator/validator/pkgs/container/validator/versions

For example, you can do:

```
docker pull \
  ghcr.io/validator/validator@sha256:27059bded1fe337e56faea4eaafe110ff67ba501daf1555f6c11ba5655831f41
```

**Comment by GergelyKalmar** [comment]
Good point, I forgot we can just go the SHA256 route. We'll do that, thank you again for this feature!
